omod-2.2/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_2/PersonController2_2Test.java

```diff
@@ -75,7 +75,15 @@ public void shouldEditAPerson() throws Exception {
         assertTrue(person.isDead());
         assertNotNull(person.getCauseOfDeathNonCoded());
     }
-    
+    @Test
+    public void shouldEditAPersonBirthTime() throws Exception {
+        Person person = service.getPersonByUuid(getUuid());
+        String json = "{\"birthtime\": \"1970-01-01T20:20:00.000\"}";
+        SimpleObject response = deserialize(handle(newPostRequest(getURI() + "/" + getUuid(), json)));
+        assertNotNull(response);
+        Object responsePersonContents = PropertyUtils.getProperty(response, "person");
+        assertNotNull(responsePersonContents);
```

Comment:
Can you also assert that the birthtime was updated?

Reply:
yeah, it is not giving error now during registration in bahmni-standard also test case shouldEditAPersonBirthTime() running fine.

Reply:
Can you add an assertion that the birthtime was updated to the new value?

Reply:
there is timezone difference between expected and updated birth time that's why  assertEquals("1970-01-01T20:20:00.000+0000", PropertyUtils.getProperty(responsePersonContents, "birthtime").toString()); thi line of code giving error:
org.junit.ComparisonFailure: 
Expected :1970-01-01T20:20:00.000+0000
Actual   :1975-04-08T20:20:00.000+0530

Reply:
Please check whether it is correct or not

Reply:
Does the test pass?

Reply:
@harshkumar8789 does this test pass on your local machine?

Reply:
@dkayiwa test passes successfully in my local machine.